src/neophile/repository.py

```diff
@@ -71,7 +71,10 @@ def switch_branch(self) -> None:
 
     def update(self) -> None:
         """Update an existing checkout to its current upstream."""
-        self._repo.heads["master"].checkout()
+        if "main" in self._repo.heads:
```

Comment:
This is fair. Arguably this method could query GitHub for the default branch configured for the repo, but the existing code matches what should generally be happening.